### ISSUE TITLE
Error with N:N records

### ISSUE THREAD

**Issue by lykebank** [lucasalexander/AlexanderDevelopment.ConfigDataMover#15]
Where my relationship entity name is 'tri_eventconfig_x_msgheader', I'm getting this error when trying to import intersect entities into a CRM org:

2019-02-26 16:30:08,580 RECORD ERROR: 619cc196-042f-e911-a960-001dd80081ad, tri_eventconfig_x_msgheader, OPERATION: UPDATE, MESSAGE: The Entity Relationship with SchemaName = 'tri_eventconfig_x_msgheader' was not found in the MetadataCache

I have confirmed the N:N relationship exists in the target org and that it has the same name.

**Comment by lykebank** [comment]
Found an issue in the Associate step of processing N:N imports... when trying to use the relationship entity name as the relationship name, the operation fails.

The manual workaround I found was to export to file using the relationship entity name (so the query works). In the JSON file, replace intersect records objects' "LogicalName" property value with the relationship name. And in the import job, on the step that imports N:N records, replace name attribute in entity node with relationship name, as well.
